hedera-node/hedera-mono-service/src/main/java/com/hedera/services/state/serdes/IoUtils.java

```diff
@@ -77,10 +77,10 @@ public static <T> void writeNullable(
         }
     }
 
-    public static byte[] byteStream(JKeySerializer.StreamConsumer<DataOutputStream> consumer)
+    public static byte[] byteStream(final JKeySerializer.StreamConsumer<DataOutputStream> consumer)
             throws IOException {
-        try (ByteArrayOutputStream bos = new ByteArrayOutputStream()) {
-            try (DataOutputStream dos = new DataOutputStream(bos)) {
+        try (final ByteArrayOutputStream bos = new ByteArrayOutputStream()) {
```

Comment:
Adding `final`s wherever needed, is it IntelliJ setting ?

Reply:
yes, I have activated it in IntelliJ